controllers/configurationpolicy_controller.go

```diff
@@ -1392,8 +1414,22 @@ func (r *ConfigurationPolicyReconciler) handleSingleObj(
 
 		compType := strings.ToLower(string(objectT.ComplianceType))
 		mdCompType := strings.ToLower(string(objectT.MetadataComplianceType))
+
+		before := time.Now().UTC()
+
 		throwSpecViolation, msg, pErr := r.checkAndUpdateResource(obj, compType, mdCompType, remediation, dclient)
 
+		duration := time.Now().UTC().Sub(before)
+		seconds := float64(duration) / float64(time.Second)
+		compareObjSecondsCounter.WithLabelValues(
+			obj.policy.Name,
+			fmt.Sprintf("%s.%s", obj.gvr.Resource, obj.name),
```

Comment:
I wonder if we should add the namespace here as well, if the object is namespaced... but it might not be necessary.